client/lib/abtest/active-tests.js

```diff
@@ -167,4 +167,14 @@ export default {
 		allowExistingUsers: true,
 		localeTargets: 'any',
 	},
+	reskinSignupFlow: {
+		datestamp: '20200720',
+		variations: {
+			reskinned: 50,
+			control: 50,
+		},
+		defaultVariation: 'control',
+		allowExistingUsers: true,
+		localeExceptions: [ 'en', 'es' ],
+	},
```

Comment:
Looks like something is going wrong here, because I'm not getting assigned to the test in any locale. Maybe you should use `localeExceptions ` in conjunction with `localeTargets: 'any'`.

Reply:
@niranjan-uma-shankar I don't think we need to define `localeTargets` to use `localeExceptions`. If I understand the code correctly, each of the options does not impact on the other.
https://github.com/Automattic/wp-calypso/blob/master/client/lib/abtest/index.js#L246-L253

Reply:
Okay cool. I haven't checked that part of the code, but was wondering if that might be the issue since I am not getting assigned to the test in non-EN+ES locale.

Reply:
@taggon I verified that after adding `localeTargets: 'any'`, users are now getting assigned to the A/B test. So, I pushed in this change, hope you don't mind!